Packs/TeamViewer/README.md

```diff
@@ -0,0 +1,8 @@
+This pack contains an integration to collect events automatically from Team.
+You can also use the ***teamviewer-get-events*** command to manually collect events.
+
+1. User access token scope: Event logging - Allow requesting all event logs.

```

Comment:
This part shouldn't be here and the scopes and permissions should be in the integration readme.
Consult with @ShirleyDenkberg what should be here

Reply:
The pack readme should contain the following:
-  A short paragraph connecting real-life situations to the pack use cases.
- A "What does this pack do?" section, explaining point-by-point the capabilities of the pack or the main playbook of the pack.
For an example, see: https://github.com/demisto/content/blob/master/Packs/AWS-SecurityHub/README.md